docs/resources/compute_instance.md

```diff
@@ -353,8 +377,20 @@ The `network` block supports:
 
 The `data_disks` block supports:
 
-* `type` - (Required, String, ForceNew) Specifies the ECS data disk type, which must be one of available disk types,
-  contains of *SSD*, *GPSSD* and *SAS*. Changing this creates a new instance.
+* `type` - (Required, String, ForceNew) Specifies the ECS data disk type. Changing this creates a new instance.
+
+  For details about disk types, see
+  [Disk Types and Disk Performance](https://support.huaweicloud.com/en-us/productdesc-evs/en-us_topic_0014580744.html).
+  Available options are:
+  + `SAS`: high I/O disk type.
+  + `SSD`: ultra-high I/O disk type.
+  + `GPSSD`: general purpose SSD disk type.
```

Comment:
```suggestion
  + `SAS`: high I/O disk type.
  + `SSD`: ultra-high I/O disk type.
  + `GPSSD`: general purpose SSD disk type.
```
change the first letter to upper case of the first word in the description

Reply:
done